pkg/server/sequential/sequential.go

```diff
@@ -0,0 +1,26 @@
+// Package server is used to store composition extensions for gRPC service servers only
+package server
```

Comment:
We don't need to do `pkg/server/sequential/` if package name is `server` then it should be located in `pkg/server` directory.